test/postBuilding.test.js

```diff
@@ -0,0 +1,89 @@
+const tape = require('tape');
+const supertest = require('supertest');
+
+const app = require('../server/app');
+
+tape('Testing for /report-building', t => {
```

Comment:
create two successful tests, one without an image and another with an image (always test your work!!)